test/integration/hotrestart_handoff_test.py

```diff
@@ -40,6 +40,7 @@ def random_loopback_host():
 # Ideally this would be adjusted (3x) for tsan and coverage runs, but making that
 # possible for python is outside the scope of this test.
 STARTUP_TOLERANCE_SECONDS = 10
+PARALLEL_REQUESTS = 5
```

Comment:
in the PR description it's clear that you are doing this to make the test more likely to fail due a flake, but I think it would be helpful to have that comment repeated here also. At first I thought we were trying paper over flakiness with an OR of results but you are going to AND them and *expose* the flakiness :)